packages/vuetify/src/util/helpers.ts

```diff
@@ -214,9 +214,10 @@ export function deepEqual (a: any, b: any): boolean {
   return props.every(p => deepEqual(a[p], b[p]))
 }
 
-export function getObjectValueByPath (obj: object, path: string, fallback?: any): any {
+export function getObjectValueByPath (obj: any, path: string, fallback?: any): any {
   // credit: http://stackoverflow.com/questions/6491463/accessing-nested-javascript-objects-with-string-key#comment55278413_6491621
-  if (!path || path.constructor !== String) return fallback
+  if (obj == null || !path || path.constructor !== String) return fallback
+  if (obj[path] !== undefined) return obj[path]
```

Comment:
`path in object`? depends what you want the behaviour to be for values that exist but are undefined

Reply:
imo keys with undefined as value should be treated as key not existing